### PR TITLE
Chapter1/homework/1 and2

### PR DESCRIPTION
**Pull request by jasonlhoward**
This is my first pull request.  Hopefully, I did it correctly.  Thanks.

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -5,8 +5,33 @@ namespace BootCamp.Chapter
...
         static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
+        {            
+            //Homework #1 
+            //Read name, surename, age, weight(in kg) and height(in cm) from console.
+            //1) Print all the info based on the example message below:
+            //Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm.
+            //2) Calculate and print body-mass index(BMI)
+            //3) Do 1 and 2 for another person.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Person 2 missing

**Comment by Almantask** [comment]
Well done with good commit messages

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (jasonlhoward) [reply Src/BootCamp.Chapter/Program.cs]: Oh.  I'll add that.  I actually thought I just supposed to test with different inputs.

Comment (jasonlhoward) [reply Src/BootCamp.Chapter/Program.cs]: I made the changes.  Do I need to do another pull request?

Comment (jasonlhoward) [reply Src/BootCamp.Chapter/Program.cs]: I've used GitHub before but not with other people, so this is all new to me.

**Comment by jasonlhoward** [comment]
Added 2nd person

**Pull request closed without merging**